packages/expo-cli/src/commands/apply/configureAndroidProjectAsync.ts

```diff
@@ -102,7 +102,14 @@ export default async function configureAndroidProjectAsync(projectRoot: string)
       androidManifest
     );
 
-    androidManifest = await AndroidConfig.Updates.setUpdatesConfig(exp, androidManifest, username);
+    const sdkVersion = exp.runtimeVersion ? null : getExpoSDKVersion(projectRoot, exp);
```

Comment:
if neither sdk version nor runtimeVersion is specified it would throw an error that SDK version is required, which is not exactly correct

Reply:
In this case, you don't need that 4-th argument (already commented next to setUpdatesConfig definition about it)